src/common/utils/csv-download-utils.ts

```diff
@@ -0,0 +1,80 @@
+import { GetTransactionListTypeEnum } from '@stacks/blockchain-api-client';
```

Comment:
Could you please consider adding unit tests for both functions? It would be really helpful for ensuring the quality and reliability of our codebase. Thank you!

Reply:
Actually, I wanted to add those too but I wasn't able to locate any other unit test files. Do we have any unit test case files from where I can take a reference? Or should I create one from the scratch ?